docs/standard/glossary.md

```diff
@@ -62,6 +62,10 @@ In contrast to the CLR/CoreCLR, CoreRT is not a virtual machine, which means it
 
 See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md)
 
+## cross-platform
+
+The ability to write an application that can be used on multiple different operating systems, such as Linux, Windows and iOS, without having to re-write specifically for each one. This enables code re-use and consistency between applications on different platforms.
```

Comment:
cross-platform involves not only writing but also execution. It might be better to phrase this as, "The ability to develop and execute an application..."